src/components/url-input.yml

```diff
@@ -0,0 +1,15 @@
+tooltip_youtube:
+  en: Youtube URL
+  de: Youtube URL
+tooltip_internal-public:
+  en: Internal public storage URL
+  de: Interner öffentlicher Speicher URL
+tooltip_internal-private:
+  en: Internal private storage URL
+  de: Interner privater Speicher URL
+tooltip_external:
+  en: External URL
+  de: Externer URL
+tooltip_unsupported:
+  en: Unsupported value
+  de: Nicht unterstützten Wert
```

Comment:
Also, although technically, URL should be masculine, it is used by 99,9% percent of people as feminine, meaning you should remove the "r"s in "Interner" and "Externer". And also, it's more correct to connect "URL" with the noun before by a hyphen in German.

Reply:
Indeed, in `common.yml` we write "Externe URL" 🤦‍♀️ 
But there we don't hyphenate either. I'll update in `common.yml` as well then.

Reply:
How should the German version for `tooltip_internal-private` / `tooltip_internal-public` be actually?

Reply:
> Indeed, in `common.yml` we write "Externe URL" woman_facepalming But there we don't hyphenate either. I'll update in `common.yml` as well then.

"Externe" is an adjective, so no hyphen there. Only between noun (for example "YouTube") and "URL" there should be a hyphen.

Reply:
Hey, another thought: I think it would sound easier in both languages if we drop the URL completely and simply have these tooltips:

~~~
tooltip_youtube:
  en: Youtube
  de: Youtube
tooltip_internal-public:
  en: Internal public storage
  de: Interner öffentlicher Speicher
tooltip_internal-private:
  en: Internal private storage
  de: Interner privater Speicher
tooltip_external:
  en: External
  de: Extern
tooltip_unsupported:
  en: Unsupported
  de: Nicht unterstützt
~~~

Or maybe with the word "Source: " / "Quelle: " in front of each?

What do you think?

Reply:
Perfect, will drop. Thanks!

Reply:
Done

Reply:
Cool!